build/deps/gen/dep_com_google_protobuf.bzl

```diff
@@ -0,0 +1,18 @@
+# WARNING: THIS FILE IS AUTOGENERATED BY update-deps.py DO NOT EDIT
+
+load("@//:build/http.bzl", "http_archive")
+
+TAG_NAME = "v28.2"
```

Comment:
This puts us far ahead of the protobuf version used in perfetto, V8 and the downstream repo, but the build seems to work so maybe that's just fine?

Reply:
I guess so. Looks like it is version-compatible?